### ISSUE TITLE
API output in JSON

### ISSUE THREAD

**Issue by AustinConlon** [BOINC/boinc#3726]
JSON output as an option would be great: https://boinc.berkeley.edu/trac/wiki/WebRpc.

**Comment by talregev** [comment]
It a great Idea. 
Are you developer?

**Comment by AustinConlon** [comment]
Yup, I use the current version of the API to display statistics in my own app by parsing the XML.

**Comment by talregev** [comment]
If you have time, will you join us to help in dev?
You can replace the xml for Json.

**Comment by davidpanderson** [comment]
Let's not replace the XML with JSON.
Massive code change, no significant benefit.

**Comment by RichardHaselgrove** [comment]
Agreed - not "replace". Maintain compatibility - there are too many people with an existing investment in the XML.

But why not work up the JSON as an optional alternative, if you think it has significant advantage for the broader community.

**Comment by AenBleidd** [comment]
I agree. Add as an additional output format - it's nice to have if someone want to invest his time into this task
